### DISCUSSION TITLE
Achievements issue

### DISCUSSION THREAD

**Discussion by bluecryptodev** [community/community#30901] [Other Feature Feedback, Questions, & Ideas]
I have done tasks for Achievements.
But I couldn't get the badge

**Comment by itta611** [comment]
You need to done these Achievements more than once.
see: https://github.com/Schweinepriester/github-profile-achievements#achievements

**Comment by Turtlebole** [comment]
Check if you got the achievement in your achievement page first, if it's there but not displaying yet on the profile it's possible that the achievements are not updated yet

**Comment by jfullstackdev** [comment]
I'm now seeing two

![image](https://user-images.githubusercontent.com/47092464/186943394-f6d216f5-cbfe-4182-866b-a40bef1d5e16.png)

there are different requirements for different badges

**Comment by codeperfectplus** [comment]
maybe, you missed some point
